src/field_theory/normal.lean

```diff
@@ -5,9 +5,9 @@ Authors: Kenny Lau
 -/
 
 import field_theory.adjoin
```

Comment:
Let's keep the imports in alphabetical order. (GitHub won't let me make one suggestion moving this line up...)
```suggestion
import group_theory.solvable
import field_theory.adjoin
```